assets/admin/editor-wizard/steps/course-details-step.js

```diff
@@ -54,6 +54,7 @@ const CourseDetailsStep = ( { wizardData, setWizardData } ) => {
 						value={ wizardData.title ?? '' }
 						onChange={ updateCourseTitle }
 						maxLength={ 40 }
+						autoFocus={ true } // eslint-disable-line jsx-a11y/no-autofocus
```

Comment:
Just a small improvement scope here I think - when the value of a property is invariably `true`, we can use JSX shorthand for that. For example, in this case, we can replace `autoFocus={ true }` with just `autofocus`

Reply:
Thanks! Updated the code d3c09f5f66209470d5be31633a209ec458f04eb4.